recipes/pyuvsim/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pyuvsim" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "e32e0a765e4c60b6418bb8e97a0f85bb3abb3a7a03932d67ce35922130cbbfbf" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - astropy
+    - mpi4py
+    - numpy 1.14.*
```

Comment:
Does this need to be pinned to numpy `1.14`? [The `setup.py` specifies `>=1.10`.](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/setup.py#L33)

Reply:
I pinned it because it was erroring in the build. It relies on pyuvdata, which requires numpy>1.14 and the build was failing to resolve the conflict. I can update the setup.py to reflect that requirement if I should.

Reply:
In that case I think it's fine as it is.